lib/ProductOpener/Display.pm

```diff
@@ -3851,11 +3851,21 @@ sub search_and_display_products($$$$$) {
 			}
 			$html .= <<HTML
 <ul class="button-group">
-<li><div style="font-size:1.2rem;background-color:#eeeeee;padding:0.3rem 1rem;height:2.75rem;margin:0">$html_count</div></li>
+<li>
+	<div style="font-size:1.2rem;background-color:#eeeeee;padding:0.3rem 1rem;height:2.75rem;margin:0">
+		$html_count
+	</div>
+</li>
 <li>
 <button href="#" data-dropdown="drop1" aria-controls="drop1" aria-expanded="false" class="button dropdown small">$Lang{explore_products_by}{$lc}</button>
 <ul id="drop1" data-dropdown-content class="f-dropdown" aria-hidden="true">
 HTML
+;
+			$initjs .= <<JAVASCRIPT
+\$(".formatable").each(function(index, element) {
+	element.innerHTML = Intl.NumberFormat(navigator.language).format(element.innerHTML);
```

Comment:
What happens if we are on an old browser without Intl.NumberFormat support?

Reply:
Error thrown in console and then nothing changes. I could encapsulate that in a try statement.